### PR TITLE
created: rock paper scissors game

### PR DESCRIPTION
**Pull request by mmusk**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      (cChoice === PAPER && pChoice === SCISSORS)
+    ? PLAYER_WINS
+    : COMPUTER_WINS;
+
+startGameBtn.addEventListener("click", () => {
+  if (isGameRunning) {
+    return;
+  }
+  isGameRunning = true;
+  console.log("Game is starting...");
```
Comment (Rehmatulla-Khan) [inline app.js]: Remove this @mmusk

File: app.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  const playerSelection = getPlayerChoice();
+  const computerSelection = getComputerChoice();
+  const winner = getWinner(computerSelection, playerSelection);
+  console.log(winner);
```
Comment (Rehmatulla-Khan) [inline app.js]: Remove this as well

**Review by Rehmatulla-Khan — COMMENTED** [review]
Remove console logs i will approve also can you tell me the difference between && nd || ? Nd how it works @mmusk

**Review by Tanvir04khan — APPROVED** [review]

File: app.js (hunk 1 above)
Comment (mmusk) [reply app.js]: fixed @Rehmatulla-Khan

File: app.js (hunk 2 above)
Comment (mmusk) [reply app.js]: fixed @Rehmatulla-Khan

**Comment by mmusk** [comment]
> Remove console logs i will approve also can you tell me the difference between && nd || ? Nd how it works @mmusk

&& is known as AND operator and | | is known as OR operator, both of these are logical operators. If we use && (AND operator) it will return False even if one statement/ expression returns False while, | | (OR operator) will return True even if one statement/ expression returns True.

   T && T = T                          T | | T = T
   T && F = F                          T | | F = T
   F && T = F                          F | | T = T
   F && F = F                          F | | F = F

File: index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Rock, Paper, Scissors</title>
+</head>
+<body>
+  <button id="start-game-btn">Start Game!</button>
+</body>
+<script src="app.js"></script>
```
Comment (anask601) [inline index.html]: why  are adding script tag in body

Comment (anask601) [inline index.html]: why  are adding script tag in body

Comment (mmusk) [reply index.html]: done @anask601

Comment (mmusk) [reply index.html]: done @anask601

**Review by Rehmatulla-Khan — APPROVED** [review]

**Review by anask601 — APPROVED** [review]
LGTM

**Pull request merged**